cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -72,6 +73,22 @@ prefixFlag prefix longFlag =
     Nothing -> longFlag
     Just prefix' -> prefix' <> "-" <> longFlag
 
+bounded :: forall a. (Bounded a, Integral a, Show a) => String -> Opt.ReadM a
+bounded t = Opt.eitherReader $ \s -> do
```

Comment:
I think we're not using this function? If it's not needed I think we can remove it.

Reply:
We do use it, but only in the file where it is defined. Here is one usage:

https://github.com/IntersectMBO/cardano-cli/blob/25fcdff827c111e9bbddebfc679dc144cf96dd35/cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs#L84

but there are multiple others.